### DISCUSSION TITLE
A poor confirmation system

### DISCUSSION THREAD

**Discussion by onoregleb** [community/community#144755] [GitHub Education]
### Hi everyone,

I've never seen such a poor identity verification system in my life. Firstly, you can't upload files, just take a photo. However, you can't even take a photo from the back camera of your phone (are you normal?). Secondly, there is no support. I requested access via mail linked to an educational institution, uploaded documents, but was refused (what's the problem? Think for yourself). If you offer something, do it properly, and not with shit and sticks.

**Comment by IRNCyber** [comment]
I understand your frustration with the identity verification process. It's clear that the current system has its limitations and can be frustrating for users.
Unfortunately, I can't directly influence the specific policies and procedures of the platform you're referring to. However, I can offer some general advice:
1. Provide Clear and Constructive Feedback:
 * Contact Support: Reach out to the platform's support team and provide detailed feedback about your experience.
 * Be Specific: Explain the exact issues you encountered, such as the limitations of the photo capture process and the lack of support for file uploads.
 * Suggest Improvements: Offer constructive suggestions for improvement, such as allowing file uploads or providing more flexibility in the verification process.
2. Explore Alternative Platforms:
If you're experiencing significant difficulties with the current platform, consider exploring other options that might have more user-friendly identity verification processes.
3. Engage with the Community:
 * Online Forums: Check online forums and communities to see if other users have encountered similar issues and found solutions.
 * Social Media: Share your experience on social media platforms and tag the company's official accounts.
Remember to express your frustration in a constructive manner, as this is more likely to lead to positive changes.
